src/clr-angular/forms/datepicker/datepicker-view-manager.spec.ts

```diff
@@ -35,6 +36,17 @@ export default function() {
       viewManagerService = context.getClarityProvider(ViewManagerService);
     });
 
+    it('supports clrPosition option', () => {
```

Comment:
This makes sense.

For the 2nd one, as only the setter method is added for `position` input on `clarityDirective` (ie. `clr-datepicker-view-manager`), it can test the points after `detectChanges()`.